auth-web/src/services/business-lookup.services.ts

```diff
@@ -0,0 +1,61 @@
+import { BusinessLookupResultIF } from '@/models'
+import ConfigHelper from '@/util/config-helper'
+import { SessionStorageKeys } from '@/util/constants'
+import { axios } from '@/util/http-util'
+
+/**
+ * Class that provides integration with the BusinessLookup API.
+ */
+export default class BusinessLookupServices {
```

Comment:
There is no shared component that provides this service. This file is almost exactly the same as the one in Create UI and Edit UI (except for how the URL is fetched).

Reply:
Should we make them the same? so we can refactor into bcrs-shared-components (add it to the pile haha)

Reply:
It would require some refactoring to pass in the URL, API key and account id, but it could be done. (Any thoughts on the params?)

How about if I look into that after this PR?

Reply:
If we want it to be the same code across business search / ppr / auth / entities then we should use the search composable (it's the only one that will support the functionality for all the UIs)

Reply:
for vue 3 implementation you would only have to import it, set the env var, set the specific filters (i.e. legal type etc.), use the results array it provides, and trigger the search function so it would slim down the code a lot too

Reply:
This UI isn't at Vue 3 yet, so could we use a common implementation? Or should we wait?

Reply:
I think we should wait and then sort out the common one after/during the upgrade

Reply:
New tech debt ticket is https://github.com/bcgov/entity/issues/16821.

Reply:
Travis, Kial, OK if I resolve this conversation?